interpreter/cling/.github/workflows/ci.yml

```diff
@@ -51,6 +51,14 @@ jobs:
             compiler: gcc
             version: "9"
 
+          - name: windows-2019-ninja
+            os: windows-2019
+            generator: Ninja
```

Comment:
FYI, Ninja has never been tried on Windows and I suspect it will not work...

Reply:
Thanks!